guides/common/modules/proc_using-a-synced-kickstart-repository.adoc

```diff
@@ -1,20 +1,20 @@
 [id="using-a-synced-kickstart-repository_{context}"]
-= Using a Synced Kickstart Repository for a Host's Operating System
+= Using a synced Kickstart repository for a host's operating system
```

Comment:
```suggestion
= Using a synchronized Kickstart repository for a host's operating system
```

I also lean towards "operating system of the host" vs. "host's OS".

Reply:
Fixed 'synced' and updated anchor.

Although I agree with your "leaning", I am keeping the `'s` because the title is already quite long.